experimental/dds/tree2/src/test/rebaserAxiomaticTests.ts

```diff
@@ -154,12 +157,12 @@ export function runExhaustiveComposeRebaseSuite<TContent, TChangeset>(
 							);
 							const rebaseWithoutCompose = rebaseTagged(
```

Comment:
I'm finding this test difficult to read. Can we break it up into smaller functions?